src/pipeline/pipeline.py

```diff
@@ -153,12 +159,12 @@ def db_batched_insert_sightings(session, obj_model, cid):  # pragma: no covers
         db_ingest_elapsed_time += end_time - start_time
 
 
-def db_batched_commit_sightings(session):  # pragma: no covers
+def db_batched_commit_sightings(session, last_sightings_from_file):  # pragma: no covers
```

Comment:
Giving this new keyword argument a default value (`last_sightings_from_file=False`) makes this code backwards compatible if others were using this module as a library versus just running it as a script.